examples/ui-demo/src/components/preview/CodePreview.tsx

```diff
@@ -50,9 +52,41 @@ function CodeBlock({ title, code }: { title: string; code: string }) {
   );
 }
 
-const styleCode = `import { withAccountKitUi } from "@alchemy/aa-alchemy/tailwind";
+
+function getTailwindCode(config: Config) {
+  const { ui } = config
+  // TODO: separate primaryColor for light and dark mode
+  return `import { withAccountKitUi, createColorSet } from "@alchemy/aa-alchemy/tailwind";
```

Comment:
nit: oh useful util we use in a few other places is the `dedent` package which makes it much easier to get these things formatted in code nicely. basically lets you do:
```
return dedent`
import ...

....
`;
```

Reply:
oh good idea, added!